README.md

```diff
@@ -156,10 +156,10 @@ HOF-bootstrap accepts the following options so a developer can customise element
 
 ### Options
 
-- `views`: Location of the base views relative to the root of your project. Defaults to 'views'. Set `views` to `false` if not present.
+- `views`: Location of the base views relative to the root of your project. Defaults to `__dirname + '/views'`.
+- `fields`: Location of the common fields relative to the root of your project. Defaults to `__dirname + '/fields'`.
+- `translations`: Location of the common translations relative to the root of your project. Defaults to `./translations`.
```

Comment:
without root views extension is `hof-template-partials -> your-route-views`, if you give it a common views folder it will be `hot-template-partials->your-common-views->your-route-views`